docs/lib/debugging/fragments/android/dev-menu/usage.md

```diff
@@ -0,0 +1,6 @@
+To access the developer menu during a debug build of your app, shake the device. An emulator can be 
```

Comment:
"and" shake the device.

Reply:
Or consider putting the "action first". "Shake the device to bring up the developer menu ..."